client/src/main/java/com/paypal/selion/platform/grid/SeLionSelendroidDriver.java

```diff
@@ -332,7 +332,7 @@ private void performLongClickAction(Point point) {
         }
     }
     
-    private void performSwipeAction(Point start, Point end) {
+    public void performSwipeAction(Point start, Point end) {
```

Comment:
Needs java documentation (javadoc) if the visibility is to be increased to public